frontend/src/router.js

```diff
@@ -181,6 +181,20 @@ export default new Router({
         {
           path: 'story',
           name: 'camp/story',
+          async beforeEnter (to, from, next) {
+            const period = await firstFuturePeriod(to)
+            if (period) {
+              await period.camp()._meta.load
+              next(periodRoute(period, 'camp/story/period', to.query))
+            } else {
+              const camp = await apiStore.get().camps({ campId: to.params.campId })
+              next(campRoute(camp, 'admin', to.query))
+            }
+          }
+        },
+        {
+          path: 'story/:periodId/:periodTitle?',
```

Comment:
It works well like this, but have you thought about instead appending `/story` to the period?
`/camps/abc1234/So-La/periods/1234abc/Hauptlager` would display the program of the period
`/camps/abc1234/So-La/periods/1234abc/Hauptlager/story` would display the story of the period

That might even help with keeping the currently selected period when navigating between program and story (and later probably also SiKo)